sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -194,11 +194,6 @@ namespace Azure { namespace Core { namespace Http {
      * @brief If set, integrates libcurl's internal tracing with Azure logging.
      */
     bool EnableCurlTracing = false;
-
-    /**
-     * @brief If set, enables libcurl's internal SSL session caching.
-     */
-    bool EnableCurlSslCaching = true;
```

Comment:
This is the change from #6581. The plan is to merge #6581 first, and then these lines will disappear from this PR. This PR demonstrates that CI will be successful when that happens.